modules/rup/routes/prestacion.ts

```diff
@@ -515,13 +514,20 @@ router.get('/prestaciones/:id*?', async (req, res, next) => {
     }
 });
 
-router.post('/prestaciones', (req, res, next) => {
+router.post('/prestaciones', async (req, res, next) => {
     let dto = parseDate(JSON.stringify(req.body));
+
+    if (req.body.solicitud && req.body.solicitud.turno) {
+        let prestacionExistente = await Prestacion.find({ 'solicitud.turno': req.body.solicitud.turno });
```

Comment:
Dejo una sugerencia de optimización. Evita traer todo el objeto de la prestación, ya que no se usa.
```suggestion
        const length = await Prestacion.count({ 'solicitud.turno': req.body.solicitud.turno });
        if (length > 0) { 
           ...
```